Error while try to installing orionjs:core into a fresh app

I trying to installing orionjs:core into a fresh app but it breaks and throw the following error ->

```
 => Errors while adding packages:             
                                              
While loading package npm-bcrypt@0.7.7:
error: Command failed: /home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/npm rebuild
--update-binary
../src/bcrypt_node.cc: In destructor ‘virtual {anonymous}::baton_base::~baton_base()’:
../src/bcrypt_node.cc:118:18: error: ‘class v8::Persistent<v8::Function>’ has no member named ‘Dispose’
callback.Dispose();
^
../src/bcrypt_node.cc: At global scope:
../src/bcrypt_node.cc:202:19: error: variable or field ‘GenSaltAsync’ declared void
void GenSaltAsync(uv_work_t* req) {
^
../src/bcrypt_node.cc:202:19: error: ‘uv_work_t’ was not declared in this scope
../src/bcrypt_node.cc:202:30: error: ‘req’ was not declared in this scope
void GenSaltAsync(uv_work_t* req) {
^
../src/bcrypt_node.cc:487:30: error: expected ‘}’ at end of input
NODE_MODULE(bcrypt_lib, init);
^
../src/bcrypt_node.cc:85:13: warning: ‘void crypto_lock_init()’ defined but not used [-Wunused-function]
static void crypto_lock_init(void) {
^
../src/bcrypt_node.cc:95:13: warning: ‘void crypto_lock_cb(int, int, const char*, int)’ defined but not used [-Wunused-function]
static void crypto_lock_cb(int mode, int n, const char* file, int line) {
^
../src/bcrypt_node.cc:105:22: warning: ‘long unsigned int crypto_id_cb()’ defined but not used [-Wunused-function]
static unsigned long crypto_id_cb(void) {
^
make: *** [Release/obj.target/bcrypt_lib/src/bcrypt_node.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit
(/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:12)
gyp ERR! System Linux 4.4.0-53-generic
gyp ERR! command "/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node"
"/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js"
"rebuild"
gyp ERR! cwd /home/divyanshu/.meteor/packages/npm-bcrypt/.0.7.7.7y4so1++os.linux.x86_64+web.browser+web.cordova/npm/node_modules/.temp-bl79ik/node_modules/bcrypt
gyp ERR! node -v v4.6.2
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok 

npm ERR! Linux 4.4.0-53-generic
npm ERR! argv "/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node"
"/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/npm" "rebuild" "--update-binary"
npm ERR! node v4.6.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! bcrypt@0.7.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bcrypt@0.7.7 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bcrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bcrypt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls bcrypt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/divyanshu/.meteor/packages/npm-bcrypt/.0.7.7.7y4so1++os.linux.x86_64+web.browser+web.cordova/npm/node_modules/.temp-bl79ik/npm-debug.log
../src/bcrypt_node.cc: In destructor ‘virtual {anonymous}::baton_base::~baton_base()’:
../src/bcrypt_node.cc:118:18: error: ‘class v8::Persistent<v8::Function>’ has no member named ‘Dispose’
callback.Dispose();
^
../src/bcrypt_node.cc: At global scope:
../src/bcrypt_node.cc:202:19: error: variable or field ‘GenSaltAsync’ declared void
void GenSaltAsync(uv_work_t* req) {
^
../src/bcrypt_node.cc:202:19: error: ‘uv_work_t’ was not declared in this scope
../src/bcrypt_node.cc:202:30: error: ‘req’ was not declared in this scope
void GenSaltAsync(uv_work_t* req) {
^
../src/bcrypt_node.cc:487:30: error: expected ‘}’ at end of input
NODE_MODULE(bcrypt_lib, init);
^
../src/bcrypt_node.cc:85:13: warning: ‘void crypto_lock_init()’ defined but not used [-Wunused-function]
static void crypto_lock_init(void) {
^
../src/bcrypt_node.cc:95:13: warning: ‘void crypto_lock_cb(int, int, const char*, int)’ defined but not used [-Wunused-function]
static void crypto_lock_cb(int mode, int n, const char* file, int line) {
^
../src/bcrypt_node.cc:105:22: warning: ‘long unsigned int crypto_id_cb()’ defined but not used [-Wunused-function]
static unsigned long crypto_id_cb(void) {
^
make: *** [Release/obj.target/bcrypt_lib/src/bcrypt_node.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit
(/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:12)
gyp ERR! System Linux 4.4.0-53-generic
gyp ERR! command "/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node"
"/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js"
"rebuild"
gyp ERR! cwd /home/divyanshu/.meteor/packages/npm-bcrypt/.0.7.7.7y4so1++os.linux.x86_64+web.browser+web.cordova/npm/node_modules/.temp-bl79ik/node_modules/bcrypt
gyp ERR! node -v v4.6.2
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok 

npm ERR! Linux 4.4.0-53-generic
npm ERR! argv "/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node"
"/home/divyanshu/.meteor/packages/meteor-tool/.1.4.2_3.yimmd4++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/npm" "rebuild" "--update-binary"
npm ERR! node v4.6.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! bcrypt@0.7.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bcrypt@0.7.7 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bcrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bcrypt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls bcrypt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/divyanshu/.meteor/packages/npm-bcrypt/.0.7.7.7y4so1++os.linux.x86_64+web.browser+web.cordova/npm/node_modules/.temp-bl79ik/npm-debug.log
```
Any advise or suggestion what can I do for removing this error.
Thanks.

--- a/packages/core/package.js
+++ b/packages/core/package.js
@@ -9,6 +9,7 @@ Package.onUse(function(api) {
   api.versionsFrom('1.0');
 
   api.use([
+    'npm-bcrypt@0.9.2',
     'blaze-html-templates@1.0.1',
     'ecmascript@0.1.6',
     'orionjs:base@1.8.0',

